### ISSUE TITLE
Reading CKM matrix from SLHA input file

### ISSUE THREAD

**Issue by jjonesperez** [flav-io/flavio#11]
Would be great if flavio would also read either the VCKMIN or both VCKM and IMVCKM entries in an SLHA file.

**Comment by DavidMStraub** [comment]
Good idea; this could in principle be applied to other parameters as well.

For `VCKMIN`, this is easy to do; `VCKM` and `VCKMIN` are more tricky, as they are not required to be unitary (as far as I understand). In flavio the CKM matrix is exactly unitary (deviations from unitarity can always be expressed as dimension-6 operators).

Are you thinking about a particular code? The SPheno/FlavorKit example file in the repository
https://github.com/flav-io/flavio/blob/master/flavio/data/test/SPheno.spc.MSSM
does not contain any CKM parameters.

**Comment by jjonesperez** [comment]
I guess it'd be enough if VCKMIN could be read!

I am thinking of SPheno/FlavorKit, the CKM matrix can be set as output with a switch. See, for example, this: https://www.dropbox.com/s/vv3a9rbuxa3n0kp/SPheno.spc.MSSM.1?dl=0

**Comment by DavidMStraub** [comment]
OK!

This is implemented now in dfa7f73c201e990f7d4ffd3b2417b420b288d130. It will be released with the next version (tomorrow or so). Thanks for providing the example file, I included it as a test case.

Documentation:
http://flav-io.github.io/apidoc/flavio/io/flha.m.html#flavio.io.flha.read_ckm

Usage:

```python
import flavio
from flavio.io import flha

my_wc = flha.read_wilson('SPheno.spc.MSSM')
flavio.config['implementation']['CKM matrix'] = 'Wolfenstein'
flha.read_ckm('SPheno.spc.MSSM')
```

Closing.
